src/universal/components/ListFilterSet/ListFilterSet.js

```diff
@@ -3,6 +3,8 @@ import cssModules from 'react-css-modules';
 import styles from './ListFilterSet.scss';
 
 @cssModules(styles)
+// for the decorators
+// eslint-disable-next-line react/prefer-stateless-function
```

Comment:
ahhh gotcha. eh, we'll be doing about with `react-css-modules` very soon anyways, so no harm. I wonder if react-look's HOC can handle functions, or maybe they offer a HOF alternative...